apps/system/build/build.js

```diff
@@ -146,6 +146,35 @@ SystemAppBuilder.prototype.enableFirefoxSync = function(options) {
   preprocessor.execute(options, 'FIREFOX_SYNC', fileList);
 };
 
+SystemAppBuilder.prototype.disableBluetoothFeatures = function(options) {
+  var fileList = {
+    process: [
+      ['index.html'],
+      ['js', 'quick_settings.js']
+    ],
+    remove: {
+      'ifdef': [
+        ['js', 'bluetooth.js'],
```

Comment:
Sorry for inconvenient, with Bug 1211357 landed (remove BT APIv1), some files should be removed from the list.